src/Image.svelte

```diff
@@ -1,10 +1,23 @@
 <script>
-  /**
-   * Cloudinary Image Component
-   */
+    /**
+     * Cloudinary Image Component
+     */
+    import {onMount, afterUpdate} from 'svelte';
+    import {getImageTag, responsive} from "./utils";
+    const bindImage = (node) => (imgElement = node);
 
-  import { getImageTag } from "./utils";
-  $: imageTag = getImageTag($$props);
+    let imgElement; // Reference to underlying <img> element
+    let isResponsive;
+
+    $: attributes = getImageTag($$props).attributes();
+
+    afterUpdate(() => {
+        // Run responsive() only when props.responsive is updated to truthy value
+        isResponsive = $$props.responsive && imgElement && !isResponsive;
```

Comment:
Can you explain the `!isResponsive` at the end?

Reply:
Removed this